src/com/github/mtdrewski/GRAPH_moment/controller/ImportController.java

```diff
@@ -0,0 +1,39 @@
+package com.github.mtdrewski.GRAPH_moment.controller;
+
+import com.github.mtdrewski.GRAPH_moment.model.dataProcessor.DataProcessor;
+import com.github.mtdrewski.GRAPH_moment.model.graphs.Graph;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
+import javafx.scene.control.TextArea;
+import javafx.stage.Stage;
+
+import java.io.IOException;
+
+public class ImportController {
+
+    @FXML
+    TextArea textArea;
+
+    public GraphDrawer graphDrawer;
+
+    DataProcessor dataProcessor = new DataProcessor();
+
+    public void importGraphOnClick() throws IOException {
+        String textInput = textArea.getText();
+        Graph graph;
+        try {
+            graph = dataProcessor.makeGraphFromInput(textInput, DataProcessor.Type.EDGE_LIST);
+        } catch (DataProcessor.IncorrectInputFormatException e) {
+            System.out.println(e.getMessage());         //TODO: maybe make this message as pop-up
+            return;
+        }
+        ((Stage) textArea.getScene().getWindow()).close();
+        System.out.println(graph);
+
+    }
+    public void canYouHearMe(){
```

Comment:
If this is a debug function then remove it or at least mark as TODO

Reply:
Sorry, forgot to delete

Reply:
Wait, it is removen (outdated :p)